bids/layout/layout.py

```diff
@@ -465,6 +465,13 @@ def get(self, return_type='object', target=None, extensions=None,
 
         Returns:
             A list of BIDSFiles (default) or strings (see return_type).
+
+        Notes:
+            As of pybids 0.7.0 some keywords have been changed. Namely: 'type'
+            becomes 'suffix', 'modality' becomes 'datatype', 'acq' is removed
```

Comment:
hm... `acq` was removed? I don't think so.
Indeed change of the "parsing schema" should be depicted somewhere in the docstring(s), but I wonder if `get` is the best place for that.  If anything to be described here, would be the change in 0.8.0 API (e.g. `domain` / `derivatives` -> `scope`).  Changes to the schema might need to be described at the level of the BIDSLayout itself and then only referenced here may be (just an idea)?

Reply:
`acq` is now `acquisition`. I agree that `get` is probably not the right place to document changes (except for raising warnings when old and now-invalid arguments are used). I think the right place for this would be a separate introductory doc that maps the BIDS spec onto pybids (maybe called something like "pybids for BIDS users").

Reply:
Oh, I forgot about it! see it now reflected in CHANGELOG.md. #407

Reply:
Re "pybids for BIDS users" - good idea

Reply:
@TheJoggler could you update this to say that `'acq'` is now `'acquisition'`? Thanks!

Reply:
I will update as soon as I get to my computer! Thank you!

Reply:
Was it something like this you meant?